docs/admin/kubeadm.md

```diff
@@ -820,6 +820,15 @@ This means you can, for example, prepopulate `/etc/kubernetes/pki/ca.crt`
 and `/etc/kubernetes/pki/ca.key` with an existing CA, which then will be used
 for signing the rest of the certs.
 
+## Self-hosting the Kubernetes control plane  {#self-hosting}
+As of 1.8, kubeadm can experimentally create a _self-hosted_ Kubernetes control
+plane. This means that key components such as the API server, controller
+manager, and scheduler run as normal pods instead of static pods.
+
+Self-hosting is beta in kubeadm 1.8 and is expected to become the default in
```

Comment:
I can't find anything definitive from our sig meeting notes about whether we decided to call self hosting alpha or beta in 1.8.

Reply:
We decided that this is **alpha**

Reply:
Fixed, thanks. I'm happy we decided that.

I couldn't remember either but found https://github.com/kubernetes/kubernetes/blob/8163d147ec666cea006872f5ff6a65b4f9f5b542/cmd/kubeadm/app/features/features.go#L29. We should probably fix that comment.